resources/pdns_recursor_service_sysvinit.rb

```diff
@@ -41,8 +41,17 @@
 
 
 action :enable do
-  # Some sysvint systems start the default pdns-recursor after installations
-  # We make sure to disable it which also stops the service in some platforms
+
+  # Some distros start pdns-recursor after installing it, we want to stop it
+  # The behavior of the init script on CentOS 6 causes a bug so we skip it there
+  # (see https://github.com/dnsimple/chef-pdns/issues/77#issuecomment-311644973)
+  service 'pdns-recursor' do
```

Comment:
A way around this would be something like:

```ruby
pdns_recursor_actions = [:disable]
pdns_recursor_actions = pdns_recursor_actions.unshift(:stop) if node['platform_family'] == 'debian'

service 'pdns-recursor' do
    supports restart: true, status: true
    action pdns_recursor_actions
end
```